capybara-webkit.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = ">= 1.9.0"
 
-  s.add_runtime_dependency("capybara", ">= 2.3.0", "< 2.5.0")
+  s.add_runtime_dependency("capybara", ">= 2.3.0")
```

Comment:
We only bump the version lock after we've verified compatibility. We're aware of an incompatibility with Capybara 2.5, but we plan to bump the lock regardless so that folks can get started with Capybara 2.5. 

However, I'm not comfortable with an unrestricted lock, or even ~> 2.3, as we've seen breakage in the past and do not want to expose our users to unexpected issues. 

Also, when we add compatible version, we add an Appraisal for that version. We typically support two point releases of Capybara for compatibility. This ensures we can provide support for user issues.